src/apps/frontend/pages/authentication/phone-login/phone-login-form.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import { FormControl, Input } from '../../../components';
+import { AsyncError } from '../../../types';
+
+import CountryCodeSelect from './country-code-select';
+import usePhoneLoginForm from './phone-login-form.hook';
+
+interface PhoneLoginFormProps {
+  onSuccess: () => void;
+  onError: (error: AsyncError) => void;
+}
+
+const PhoneLoginForm: React.FC<PhoneLoginFormProps> = ({ onError, onSuccess }) => {
+  const { formik, isSendOTPLoading } = usePhoneLoginForm({ onSuccess, onError });
+
+  const setFormikFieldValue = (fieldName: string, data: string) => {
+    formik.setFieldValue(fieldName, data)
+      .then().catch((err) => { onError(err as AsyncError); });
+  };
+
+  const handleChangePhone = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = target;
+    setFormikFieldValue('phoneNumber', value);
+  };
+
+  const handleChangeSelect = ({ target }: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = target;
+    const [countryCode, country] = value.split(',');
+    setFormikFieldValue('country', country);
+    setFormikFieldValue('countryCode', countryCode);
+    setFormikFieldValue('phoneNumber', '');
+  };
+
+  return (
+      <>
+        <h2 className="mb-9 text-2xl font-bold text-black dark:text-white sm:text-title-xl2">
+          Log In
+        </h2>
+
+        <form onSubmit={formik.handleSubmit}>
+          <div className="mb-6">
+            <FormControl
+              label={'Phone'}
+              error={formik.touched.phoneNumber && formik.errors.phoneNumber}
+            >
+              <div className="flex gap-5">
+                <CountryCodeSelect
+                  isLoading={isSendOTPLoading}
+                  value={[formik.values.countryCode, formik.values.country]}
+                  handleChange={handleChangeSelect}
+                />
+                <Input
+                  data-testid="phoneNumber"
+                  disabled={isSendOTPLoading}
+                  error={formik.touched.phoneNumber && formik.errors.phoneNumber}
+                  name="phoneNumber"
+                  onChange={handleChangePhone}
+                  onBlur={formik.handleBlur}
+                  placeholder="Enter your phone number"
+                  type='number'
+                  value={formik.values.phoneNumber}
+                />
+              </div>
+            </FormControl>
+          </div>
+
+          <button
```

Comment:
try to create a global primary button (with the same code as [here](https://github.com/jalantechnologies/boilerplate-mern/blob/4c59df52cd624877d1d76c25b09cbee91cb872f0/src/apps/frontend/components/button/button-primary.tsx) to avoid conflicts), and use it in the components
Note: spinner for loading state is not added yet in that button component